modules/sqs/main.tf

```diff
@@ -0,0 +1,74 @@
+## Terraform Requirements
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"
+    }
+  }
+}
+
+
+## AWS Provider Settings
+provider "aws" {
+  region = var.region
+}
+
+## Local Variables
+locals {
+  tags = merge(var.tags, { Deployment = var.prefix })
+}
+
+## copy_files_to_archive_get_message_queue - SQS queue that copy_files_to_archive lambda writes to
+## ====================================================================================================
+resource "aws_sqs_queue" "copy_files_to_archive_get_message_queue" {
+  ## OPTIONAL
+  name                      = "${var.prefix}-get-message-queue"
```

Comment:
Wonder if we should adjust the naming to `staged-recovery-queue` or some other name that better denotes that messages in this queue are staged and ready for recovery to do something with. In our case we copying back to the Cumulus primary location but an end user could replace our `copy_to_archive` lambda with their own lambda to perform some other function for recovery once the data is staged from glacier and available in S3 again.

Reply:
changed the name to "${var.prefix}-staged-recovery-queue